Create landing_page.md (#1501)
dbt-labs/docs.getdbt.com: website/docs/faqs/landing_page.md

> Our [Frequently Asked Questions](https://docs.getdbt.com/faqs/all) (FAQs) section is a space where you can find an answer to some questions we get asked a lot (but that we’re happy to answer!). 

Reviewer: ```suggestion
In these frequently asked questions, you will find answers to common questions. 
```

> If you have a question or are still stuck on something, just reach out to us by emailing [support@getdbt.com](mailto:support@getdbt.com) or clicking on the chat widget, and we'll do our best to help out.

Reviewer: ```suggestion
If you have a question or are still stuck on something, you can always reach out to support by emailing [support@getdbt.com](mailto:support@getdbt.com) or using the chat widget in dbt Cloud, and we'll do our best to help out.
```

### What is dbt?

> dbt (data build tool) enables analytics engineers to transform data in their warehouses by simply writing select statements. dbt handles turning these select statements into tables and views.

Reviewer: ```suggestion
dbt enables analytics engineers to transform data in their warehouses by simply writing select statements. dbt handles turning these select statements into tables and views.
```

In these frequently asked questions, you will find answers to common questions. 

> If you have a question or are still stuck on something, you can always reach out to support by emailing [support@getdbt.com](mailto:support@getdbt.com) or using the chat widget in dbt Cloud, and we'll do our best to help out.

Reviewer: ```suggestion
If you have a question or need clarity, you can always reach out to support by emailing [support@getdbt.com](mailto:support@getdbt.com) or using the chat widget in dbt Cloud, and we'll do our best to help out.
```
